tests/integration/docusaurus/reference/core_concepts/checkpoints_and_actions.py

```diff
@@ -0,0 +1,434 @@
+"""
+context.add_checkpoint(**yaml.load(no_nesting))
+results = context.run_checkpoint(checkpoint_name="my_checkpoint")
+assert results.success == True
+
+nesting_with_defaults = """
+name: my_checkpoint
+config_version: 1
+class_name: Checkpoint
+run_name_template: "%Y-%M-foo-bar-template-$VAR"
+validations:
+  - batch_request:
+      datasource_name: taxi_datasource
+      data_connector_name: default_inferred_data_connector_name
+      data_asset_name: yellow_tripdata_sample_2019-01
+      data_connector_query:
+        index: -1
+  - batch_request:
+      datasource_name: taxi_datasource
+      data_connector_name: default_inferred_data_connector_name
+      data_asset_name: yellow_tripdata_sample_2019-02
+      data_connector_query:
+        index: -1
+expectation_suite_name: my_expectation_suite
+action_list:
+  - name: store_validation_result
+    action:
+      class_name: StoreValidationResultAction
+  - name: store_evaluation_params
+    action:
+      class_name: StoreEvaluationParametersAction
+  - name: update_data_docs
+    action:
+      class_name: UpdateDataDocsAction
+evaluation_parameters:
+  param1: $MY_PARAM
+  param2: 1 + $OLD_PARAM
+runtime_configuration:
+  result_format:
+    result_format: BASIC
+    partial_unexpected_count: 20
+"""
+context.add_checkpoint(**yaml.load(nesting_with_defaults))
+results = context.run_checkpoint(checkpoint_name="my_checkpoint")
+assert results.success == True
+first_validation_result = list(results.run_results.items())[0][1]["validation_result"]
+second_validation_result = list(results.run_results.items())[1][1]["validation_result"]
+
+first_expectation_suite = first_validation_result["meta"]["expectation_suite_name"]
+first_data_asset = first_validation_result["meta"]["active_batch_definition"][
+    "data_asset_name"
+]
+second_expectation_suite = second_validation_result["meta"]["expectation_suite_name"]
+second_data_asset = second_validation_result["meta"]["active_batch_definition"][
+    "data_asset_name"
+]
+assert first_expectation_suite == "my_expectation_suite"
+assert first_data_asset == "yellow_tripdata_sample_2019-01"
+assert second_expectation_suite == "my_expectation_suite"
+assert second_data_asset == "yellow_tripdata_sample_2019-02"
+documentation_results = """
+print(first_expectation_suite)
+my_expectation_suite
+
+print(first_data_asset)
+yellow_tripdata_sample_2019-01
+
+print(second_expectation_suite)
+my_expectation_suite
+
+print(second_data_asset)
+yellow_tripdata_sample_2019-02
+"""
+
+
+keys_passed_at_runtime = """
+name: my_base_checkpoint
+config_version: 1
+class_name: Checkpoint
+run_name_template: "%Y-%M-foo-bar-template-$VAR"
+action_list:
+  - name: store_validation_result
+    action:
+      class_name: StoreValidationResultAction
+  - name: store_evaluation_params
+    action:
+      class_name: StoreEvaluationParametersAction
+  - name: update_data_docs
+    action:
+      class_name: UpdateDataDocsAction
+evaluation_parameters:
```

Comment:
Is there a way to assert that the evaluation parameters as configured worked as expected?